app/controllers/auth_controller.rb

```diff
@@ -1,5 +1,5 @@
 class AuthController < ApplicationController
-  skip_before_action :authenticate_request, only: %i[login register verify_email generate_otp verify_otp] 
+  skip_before_action :authenticate_request, only: %i[login register verify_email generate_otp verify_otp delete_user] 
```

Comment:
Anyone can delete anyone's account now, Which should not be allowed. Only the authenticated user can delete his account

Reply:
Yes. I also was concerned about that.
So, it must be like:
- passing email & password
- and if authorized then delete.
Will it be a good implementation?